### ISSUE TITLE
Can I start new conversation with a file?

### ISSUE THREAD

**Issue by sdimantsd** [AshwinPathi/claude-api-py#7]
I see an option to create new conversation and in the second message upload a file. Can I upload a file in the a new conversation?

**Comment by AshwinPathi** [comment]
Could you give an example of an API you want? Would it be something like:

```py
# Current API: 
# start_new_conversation(conversation_name: str, initial_message: str)
conversation_uuid = claude_obj.start_new_conversation("New Conversation", "Hi Claude!")

# Proposed change:
# start_new_conversation(conversation_name: str, initial_message: str, attachments: List[Attachment])
conversation_uuid = claude_obj.start_new_conversation("New Conversation", "Hi Claude!", attachments = [...])
```

**Comment by sdimantsd** [comment]
Yes, that will be grate! Thanks

**Comment by AshwinPathi** [comment]
@sdimantsd  I just added an additional argument to the `start_new_conversation` method called `attachments`. It accepts a list of `AttachmentType` objects to send.

You can create an `AttachmentType` object by calling `get_attachment(file_path)` from the client. Let me know if it works

**Comment by sdimantsd** [comment]
Thanks!
How can I get it?
Its in pip?
How can I install from git?

**Comment by AshwinPathi** [comment]
You should be able to use `pip install git+git://github.com/AshwinPathi/claude-api-py.git` to install the git repo as a package.

**Comment by sdimantsd** [comment]
Thanks

**Comment by sdimantsd** [comment]
How can I get the response using this method?  
The function return the uuid, not the output

**Comment by AshwinPathi** [comment]
@sdimantsd  updated. The `start_new_conversation()` method will return a _**JSON**_ with the following contents:
```py
return {
    'uuid': {conversation_uuid_string},
    'title': {conversation_title_string},
    'response': {initial_response_string}
}
```

**Comment by sdimantsd** [comment]
Greate Thanks

**Comment by sdimantsd** [comment]
Can you return also the ['stop_reason']?

**Comment by AshwinPathi** [comment]
@sdimantsd  tbh i'll just return the entire JSON response from the send message in the `response` field if that sounds more reasonable.

**Comment by sdimantsd** [comment]
Thanks :-D
I hope I will try in a few hours.

**Comment by winie-hy** [comment]
Hello, if you want to improve the speed of any optimization suggestions author, when I input a text content of thousands of words, it takes 80s time to answer, is it because I opened too many conversations? Do I need to delete the old after creating a session after completion, look forward to and thank you for your reply

**Comment by AshwinPathi** [comment]
@winie-hy thats probably a better question to ask in another issue, but its kind of expected that the API will be slow if you are sending a large prompt and receiving and equally large prompt.

The Claude API sends responses to your message in "chunks" which are streamed via Server Side Events to the client. The entire message is completely streamed only after all the chunks are receieved, which for large messages, can take a while. My API isn't asynchronous, so I basically block (wait) until all the chunks are received, and then I return the entire response to the client.

**Comment by AshwinPathi** [comment]
@sdimantsd were you able to use the new api? if so I'll close this issue.

**Comment by sdimantsd** [comment]
That works great!
Thanks
